projects/observability/src/shared/dashboard/data/graphql/waterfall/api-trace-waterfall-data-source.model.ts

```diff
@@ -100,6 +101,17 @@ export class ApiTraceWaterfallDataSourceModel extends GraphQlDataSourceModel<Wat
     return spans.map(span => this.constructWaterfallData(span, trace, duration));
   }
 
+  private mapepdLogEvents(logEventsObject: Dictionary<LogEvent[]>, startTime: number): LogEvent[] {
```

Comment:
typo in method name.

Also, could we use a better name? What do we mean by `mapped`?

Reply:
Similar code as before. Anyway we can share these?

Reply:
May be through a `logEventsService`

Reply:
Done!!